server/database.py

```diff
@@ -79,6 +79,16 @@ def create_subscriber(db, tag_id, subscriber_id):
     return ({"Success": "Resource Created"}, 201)
 
 
+# (IP) remove_subscriber removes a subscriber from a selected tag id in a database
+def remove_subscriber_from_tag(db, tag_id, subscriber_id):
+    query_results = db.tags.update_one(
+        {"snowflake_id": tag_id}, {"$pull": {"subscribers": subscriber_id}}
+    )
+    if query_results.modified_count == 0:
+        return ({"Error": "Subscriber Does Not Exist"}, 404)
+    return ({"Success": "Delete Successful"}, 200)
+
+
```

Comment:
where should we expect to wrap our error handling?

Reply:
I might be misinterpreting, but currently this follows the format for the other operations in database.py it provides an error code if there is no change to the subscriber list -> which means we either didn't successfully fetch the list or  some other connection error.

Reply:
Correct. The way we currently have things configured, functions in database.py act as queries to the db. The query functions return a status code and data if data is to be expected i.e. a Read or GET query. What to do with that data is then the responsibility of the flask server.